toolkit/scripts/toolchain/check_manifests.sh

```diff
@@ -4,12 +4,52 @@
 
 set -e
 
-TOOLCHAIN_SPEC_LIST=$1
-SPECS_DIR=$2
-MANIFESTS_DIR=$3
-DIST_TAG=$4
-DISTRO_MACRO=$5
-ARCH=$6
+help () {
+    echo "Script to check that the manifest files only contain RPMs that could have been generated from toolchain specs"
+    echo "Usage:"
+    echo "[REQUIRED] -a -> architecture to run for (x86_64 or aarch64)"
+    echo "[OPTIONAL] -t -> space-separated list of toolchain specs"
+    echo "[OPTIONAL] -s -> SPECS_DIR"
+    echo "[OPTIONAL] -m -> TOOLCHAIN_MANIFESTS_DIR"
+    echo "[OPTIONAL] -d -> DIST_TAG"
+    echo "[OPTIONAL] -i -> DISTRO_MACRO"
+    echo "[OPTIONAL] -h -> print this help dialogue and exit"
+}
+
+# assign default value
+TOOLCHAIN_SPEC_LIST=$(make --no-print-directory -s printvar-toolchain_spec_list  2> /dev/null)
```

Comment:
So, the `Makefile` is dependent on this script and this script is dependent on the `Makefile`? I can't say I'm particularly a fan of interdependencies like this, even if it does "work".

Would it make sense to add variants of `check-aarch64-manifests` and `check-x86_64-manifests` that don't include dependency lists?